internal/metadata/clusters/ibm-openshift.yaml

```diff
@@ -213,38 +213,38 @@ spec:
     ResourceQuota:
       - v1
     Role:
-      - authorization.openshift.io/v1
       - rbac.authorization.k8s.io/v1
       - rbac.authorization.k8s.io/v1beta1
-    RoleBinding:
       - authorization.openshift.io/v1
+    RoleBinding:
       - rbac.authorization.k8s.io/v1
       - rbac.authorization.k8s.io/v1beta1
+      - authorization.openshift.io/v1
     RoleBindingRestriction:
       - authorization.openshift.io/v1
     Route:
       - route.openshift.io/v1
     Scale:
-      - apps.openshift.io/v1
       - apps/v1
       - apps/v1beta1
       - apps/v1beta2
       - extensions/v1beta1
       - v1
```

Comment:
shouldn't v1 be at the top?

Reply:
No. That v1 is from before groups concept was started.